src/Aspire.Dashboard/DashboardWebApplication.cs

```diff
@@ -83,6 +83,11 @@ public DashboardWebApplication(Action<WebApplicationBuilder>? configureBuilder =
         builder.Logging.AddFilter("Microsoft.AspNetCore.Server.Kestrel", LogLevel.Error);
 #endif
 
+        // Don't log routine dashboard HTTP request info or static file access
+        builder.Logging.AddFilter("Microsoft.AspNetCore.Hosting.Diagnostics", LogLevel.Warning);
```

Comment:
This looks redundant in non-DEBUG builds. You can see on line 81 that a Warning filter is added for `Microsoft.AspNetCore`. That applies to any logs that start with that name.

I suggest putting this in an `#else` with the above's `#if !DEBUG` block.